wiki-set-up/wiki-basics.md

```diff
@@ -0,0 +1,14 @@
+# Wiki Basics
+
+## Brief Walkthrough of Local Copy for Members
+
+1. Go to the team GitHub page and click on "wiki"
```

Comment:
It's ok to leave this as-is, but you can also just use 1. for each of the points and it will auto-increase when it renders.